### PR TITLE
Show file size in the success message

### PR DESCRIPTION
**Pull request by ernilambar**
Fixes https://github.com/wp-cli/dist-archive-command/issues/100

### CODE REVIEW

**Comment by ernilambar** [comment]
@swissspidy Regarding feature tests here, should we change `should be` to `should contain` here? I would prefer to keep `should be` with whole output with size also. WDYT?

**Comment by ernilambar** [comment]
Related https://github.com/wp-cli/wp-cli/issues/5978

File: src/Dist_Archive_Command.php [hunk 1]
```diff
@@ -513,4 +515,82 @@ private function get_file_list( $source_dir_path, $excluded = false ) {
...
 	}
+
+	/**
+	 * Converts a number of bytes to the largest unit the bytes will fit into.
+	 *
+	 * @param int $bytes    Number of bytes.
+	 * @param int $decimals Precision of number of decimal places.
+	 * @return string Number string.
+	 */
+	private function get_size_format( $bytes, $decimals = 0 ) {
```
Comment (swissspidy) [inline src/Dist_Archive_Command.php:526]: As per my issue comment, we should consider contributing this to the main WP-CLI utils

**Comment by swissspidy** [comment]
> Regarding feature tests here, should we change `should be` to `should contain` here? I would prefer to keep `should be` with whole output with size also. WDYT?

`should be` is probably too strict because there can be slight differences in file sizes on different environments.

I'd go for option 3: regex

Example (untested):

```
Then STDOUT should match /^Success: Created bar8.tar.gz \(Size: \d+ B\)$/

File: src/Dist_Archive_Command.php (hunk 1 above)
Comment (swissspidy) [reply src/Dist_Archive_Command.php:526]: OK I see you opened an issue for that now.

Comment (ernilambar) [reply src/Dist_Archive_Command.php:526]: This is pretty basic version for size format. Code mentioned in the DB is pretty much tied to database arguments. So, when we add this method in Utils, there would more arguments and customization.

Comment (ernilambar) [reply src/Dist_Archive_Command.php:526]: Should we work on that before merging this PR? Or keep this here now? This is private method so, I guess it wont be difficult to replace here when main function in merged in core repo.

Comment (swissspidy) [reply src/Dist_Archive_Command.php:526]: Fair, let's keep it as-is for now.

**Comment by davidperezgar** [comment]
Thanks for the PR!

**Review by swissspidy — APPROVED** [review]

**Pull request merged**